### DISCUSSION TITLE
I need use grafana to display some data from skywalking

### DISCUSSION THREAD

**Discussion by sjfbjs** [apache/skywalking#8288] [Q&A]
Is there any way  can convert skywalking data to mysql use  skywalking api?
Nowadys, i want to display some important data like slow endpoint  or some exception from skywalking, i choose grafana as always we do .
But  i  cann't find some docs   about this . 
I'   confused.
Is there anybody can help us?  .

**Comment by wu-sheng** [comment]
Why we should? This seems very un-open-source words. Why what you want matters to the community? This is not polite.
You should learn how you should talk to a community.

**Reply by sjfbjs** [reply]
I'm sorry if I upset you .  I'm new  here , maybe I choose the wrong category .

**Reply by wu-sheng** [reply]
The thing is about how you ask and why you think this should be answered or should have/haven't. Open source community doesn't have obligation to finish your own and paid works. This isn't our duty.

**Reply by wu-sheng** [reply]
Grafana has nothing related to SkyWalking, what you want to integrate, feel free to do privately or contribute to open source.

**Reply by sjfbjs** [reply]
Ok,thanks . I  konw .   But  I started this discussion , now I just want to discuss about it  .

**Reply by sjfbjs** [reply]
Yes, you're right . Grafana has nothing related to SkyWalking.  
I'm here to ask for help, not to discuss SkyWalking's relationship with Grafana.

**Reply by wu-sheng** [reply]
Simple answer is, we don't support that.

**Comment by tisonkun** [comment]
@sjfbjs you could learn some basic English conventions to express yourself. "We want to", "as always we do“, "we didn't find", and thus "we're confused", even "we think it's very normal requirement" is full of pride (arrogance).

**Reply by sjfbjs** [reply]
I'm sorry. I didn't mean to offend you . I just want to talk about the way  to   present data from skywalking use grafana .
